services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -106,7 +106,14 @@ def get_dynamic_sidecar_spec(
             "Source": "/var/run/docker.sock",
             "Target": "/var/run/docker.sock",
             "Type": "bind",
-        }
+        },
```

Comment:
``get_dynamic_sidecar_spec`` aims to compose the request body for service creation
https://docs.docker.com/engine/api/v1.41/#tag/Service/operation/ServiceCreate

I suggest:
- add link ``SEE https://docs.docker.com/engine/api/v1.41/#tag/Service/operation/ServiceCreate`` in doc for fast reference of details
-